cmd/hyperkube/main.go

```diff
@@ -36,6 +36,7 @@ func main() {
 	hk.AddServer(NewKubectlServer())
 	hk.AddServer(NewKubeAPIServer())
 	hk.AddServer(NewKubeControllerManager())
+	hk.AddServer(NewCloudControllerManager())
```

Comment:
seems already added as 'hk.AddAlphaServer(NewCloudControllerManager())' in this file?

Reply:
oops! bad merge. thanks!

Reply:
i have updated the local-up-cluster.sh too with the "alpha" syntax